DPG/Tasks/AOTTrack/qaEfficiency.cxx

```diff
@@ -850,7 +1026,25 @@ struct QaEfficiency {
     static_assert(charge == 0 || charge == 1);
     static_assert(id > 0 || id < nSpecies);
 
-    // Selecting a specific PDGs
+    if constexpr (PDGs[id] == 0) { // All PDGs
+      if constexpr (charge == 0) {
+        for (int i = 0; i < nSpecies - 1; i++) {
+          if (PDGCfgs[i] && mcParticle.pdgCode() == PDGs[i]) {
```

Comment:
I would remove this, as it can be computed from the single efficiencies of the PDGs offline

Reply:
Ok, I will adjust.